src/state/query_hooks/useTimeline.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { useQuery } from 'react-query';
+
+// ⬇️ --> USAGE: in react component <-- ⬇️
+// const incidentsQuery = useIncident();
```

Comment:
If no longer needed, remove when you get a chance

Reply:
Took out the comments, I opened a new branch to start the modal, so the comments will be removed once I start pushing that code up.